app/controllers/site.rb

```diff
@@ -21,17 +21,26 @@ class Site < Base
 						end
 					end.compact
 
-					slim :index, locals: { sites: policy_scope(Tint::Site), git_providers: git_providers }
+					sites = policy_scope(Tint::Site)
+					repos = sites.map { |site| site.remote }
+
+					slim :index, locals: { sites: sites, repos: repos, git_providers: git_providers }
 				end
 			end
 
 			post "/" do
 				authorize Tint::Site, :create?
 
+				if params[:provider] == "github"
+					identity = DB[:identities][user_id: pundit_user[:user_id], provider: "github"]
```

Comment:
Are we accessing `user_id`?  That's how we query it.  We do use provider to select the class to use, but no JSON decode or deep-key stuff there.  It's just `identity[:provider]`